README.md

```diff
@@ -1,12 +1,13 @@
 # WordVecSpace
 A high performance pure python module that helps in loading and performing operations on word vector spaces created using Google's Word2vec tool.
 
-This module has ability to the load data into memory using `WordVecSpaceMem` and it can also support performing operations on the data which is on the disk using `WordVecSpaceAnnoy` and `WordVecSpaceDisk`.
+This module has ability to the load data into memory using `WordVecSpaceMem` and it can also support performing operations on the data which is on the disk using `WordVecSpaceAnnoy` and                   `WordVecSpaceDisk`.
 
 ## Installation
-> Prerequisites: >=Python3.5.2
+> Prerequisites: Python3.5.2
```

Comment:
Why not Prerequisites: >=Python3.5.2,  so that people will understand that it will work for above python3.5.2

Reply:
it does not work for python 3.6